docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -142,6 +143,13 @@ These are stored alongside the code scanning suites with names of the form: ``<l
 For information about creating custom query suites, see ":doc:`Creating
 CodeQL query suites <creating-codeql-query-suites>`."
 
+Diagnostic and summary information
+..................................
+
+The code scanning query suites include additional queries to report diagnostic data generated by the extractor during database creation and summary metrics for the CodeQL database generated. When the database analysis command completes, the CLI generates the results file and reports any diagnostic and summary data to standard output. If you choose to generate SARIF output, the additional data is also included in the SARIF file.
```

Comment:
I had to read the long first sentence twice. I think it would benefit from a comma after "during database creation".

Reply:
Sentence revised.